index.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
         'vendor/ember-cli-mocha/test-loader.js'
       ];
 
-      var addonOptions = app.options['ember-cli-mocha'];
+      var addonOptions = app.options['ember-cli-mocha'] || {};
       if (addonOptions && !addonOptions.disableContainerStyles) {
```

Comment:
Scratch my comment above - I re-read the conditional and see that we need this by default. Sorry!